contrib/test_jenkins.sh

```diff
@@ -892,71 +895,82 @@ run_malloc_hook_gtest() {
 			make -C test/gtest test
 }
 
-#
-# Run the test suite (gtest)
-# Arguments: <compiler-name> [configure-flags]
-#
-run_gtest() {
-	compiler_name=$1
-
+set_common_test_flags() {
 	export GTEST_RANDOM_SEED=0
 	export GTEST_SHUFFLE=1
 	# Run UCT tests for TCP over fastest device only
 	export GTEST_UCT_TCP_FASTEST_DEV=1
 	export OMP_NUM_THREADS=4
+}
 
-	# Run specific tests
-	do_distributed_task 1 4 run_malloc_hook_gtest
-	do_distributed_task 2 4 run_gtest_watchdog_test 5 60 300
-	do_distributed_task 3 4 test_memtrack
-
+set_gtest_test_flags() {
```

Comment:
you can call `set_common_test_flags` from here, since this func is never used alone

Reply:
Done